### PR TITLE
[KOGITO-3361] Fix Kogito for PMML native 

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @jiripetrlik

See https://issues.redhat.com/browse/KOGITO-3361

This PR add NativeImageTests.
Depends on kiegroup/kogito-runtimes#771

### CODE REVIEW

File: pmml-quarkus-example/pom.xml [hunk 1]
```diff
@@ -106,6 +106,9 @@
                 <goals>
                   <goal>native-image</goal>
                 </goals>
+                <configuration>
+                  <reportErrorsAtRuntime>true</reportErrorsAtRuntime>
```
Comment (evacchi) [inline pmml-quarkus-example/pom.xml]: is this still required? this usually means that there are still some issues with native image

Comment (gitgabrio) [reply pmml-quarkus-example/pom.xml]: @evacchi 
removed :+1:

**Review by danielezonca — APPROVED** [review]
LGTM :+1:

**Review by jiripetrlik — APPROVED** [review]

**Comment by gitgabrio** [comment]
@evacchi 
could you pls take a look at this and - evemtually, approve it ? Thx!

**Review by evacchi — APPROVED** [review]

**Pull request merged**